packages/grafana-data/src/dataframe/processDataFrame.ts

```diff
@@ -26,12 +27,12 @@ import { getFieldDisplayName } from '../field/fieldState';
 
 function convertTableToDataFrame(table: TableData): DataFrame {
   const fields = table.columns.map(c => {
-    const { text, ...disp } = c;
+    const { text, type, ...disp } = c as MutableColumn;
```

Comment:
rather than expose MutableColum... can we just use `any` here?

I would rather not suggest that anyone use MutableColumn :)

Reply:
Sure, no problem